content/en/docs/shared_services/supportapi/formats/create_case_req.md

```diff
@@ -11,13 +11,15 @@ description: Request format for the Create Case method.
 | Property Name | Data Type                     | Optional | Description |
 |---------------|-------------------------------|----------|-------------|
 | sac           | string                        |       no | Support access code. |
+| contact       | string                        |       no | The e-mail of the contact. The 'contact' property is required for requests that use the [OAuthApplication security scheme](https://sphereapi.admin.axway.com/sphere/api/v1/). |
```

Comment:
Just a general comment, does not require any action, IMO:
It would be good if we switch over from quoted property names to markdown code formatted property names. That is, from 'contact' to `contact`.